articles/protocols/oauth2/index.md

```diff
@@ -1,53 +1,53 @@
 ---
 description: What is the OAuth 2.0 Authorization Framework and how it works.
+toc: true
 ---
 # OAuth 2.0
 
-[OAuth 2.0](https://oauth.net/2/) is a protocol that allows a user to grant limited access to their resources on one site, to another site. This is done without the users having to expose their credentials. According to [OAuth‘s website](http://oauth.net/about/) the protocol is not unlike a valet key.
+[OAuth 2.0](https://oauth.net/2/) is a protocol that allows a user to grant limited access to their resources on one site, to another site, without having to expose their credentials.
 
-> Many luxury cars today come with a valet key. It is a special key you give the parking attendant and unlike your regular key, will not allow the car to drive more than a mile or two. Some valet keys will not open the trunk, while others will block access to your onboard cell phone address book. Regardless of what restrictions the valet key imposes, the idea is very clever. You give someone limited access to your car with a special key, while using your regular key to unlock everything.
-
-The protocol's purpose is to provide a standard way for developers to offer a service via an API without forcing the users to expose their passwords.
-
-
-## OAuth Roles
+According to [OAuth‘s website](http://oauth.net/about/) the protocol is not unlike a valet key.
 
-The [OAuth 2.0 Authorization Framework specification](https://tools.ietf.org/html/rfc6749) defines a set of roles.
+> Many luxury cars today come with a valet key. It is a special key you give the parking attendant and unlike your regular key, will not allow the car to drive more than a mile or two. Some valet keys will not open the trunk, while others will block access to your onboard cell phone address book. Regardless of what restrictions the valet key imposes, the idea is very clever. You give someone limited access to your car with a special key, while using your regular key to unlock everything.
 
-- **Resource Owner**: the entity that can grant access to a protected resource. Typically this is the end-user.
+To get access to the protected resources OAuth 2.0 uses **access tokens**. An access token is a string in [JSON Web Token (JWT)](/jwt) format, that contains the list of authorized permissions.
```

Comment:
Token format is out of OAuth 2.0 specification and its an implementation detail.